tests/unit/core/reducers/test_addons.js

```diff
@@ -251,26 +247,6 @@ describe(__filename, () => {
     });
   });
 
-  it('does not use description from theme_data', () => {
```

Comment:
does this mean we ARE using the description from theme_data now?

Reply:
yes, for LWT themes

Reply:
Should we keep this as test then, to show that it's using the right description?

Reply:
hum, no because this is not relevant anymore. Yet, the test case above should have a new description... From: `stores a modified theme object` to `stores the original theme object`